### PR TITLE
[Do Not Merge] Image Resolution Tests

### PR DESCRIPTION
**Pull request by youriwims**
Just testing Wagtail’s JPEG image quality setting to see if it'd be a good option for switching between image resolutions on various displays.

### CODE REVIEW

File: network-api/networkapi/wagtailpages/templates/wagtailpages/homepage.html [hunk 1]
```diff
@@ -14,7 +14,7 @@
   <div class="row mb-5 mt-0 mt-sm-5 d-flex align-items-center flex-sm-row-reverse homepage-feature">
     <div class="col-sm-6 mb-3 mb-sm-0">
         {% image page.hero_image max-200x200 as image_small %}
-        {% image page.hero_image max-400x400 as image_medium %}
+        {% image page.hero_image max-400x400 jpegquality-40 as image_medium %}
```
Comment (youriwims) [inline network-api/networkapi/wagtailpages/templates/wagtailpages/homepage.html]: @patjouk I added the jpegquality setting here and the quality difference is visibly apparent.

Comment (youriwims) [inline network-api/networkapi/wagtailpages/templates/wagtailpages/homepage.html]: @patjouk I added the jpegquality setting here just to see exactly what would happen and the quality difference is visibly apparent.

File: network-api/networkapi/wagtailpages/templates/wagtailpages/homepage.html [hunk 2]
```diff
@@ -15,7 +15,7 @@
     <div class="col-sm-6 mb-3 mb-sm-0">
         {% image page.hero_image max-200x200 as image_small %}
         {% image page.hero_image fill-300x169 format-jpeg jpegquality-40 as image_medium %}
-        {% image page.hero_image fill-480x270 format-jpeg as image_large %}
+        {% image page.hero_image fill-480x270-c0 format-jpeg as image_large %}
```
Comment (youriwims) [inline network-api/networkapi/wagtailpages/templates/wagtailpages/homepage.html]: The focal point needs to be around primary/important text so it's not cut off.

**Comment by Pomax** [comment]
is this a PR intended to be landed, or purely for testing?

**Comment by youriwims** [comment]
This is purely for testing.

**Pull request closed without merging**